### ISSUE TITLE
Facade custom options not being passed to Producer object

### ISSUE THREAD

**Issue by renatobcj** [mateusjunges/laravel-kafka#13]
Im trying to instantiate a producer with a remote kafka broker (confluent), but the custom options that im setting on the Kafka facade are not going to the final Producer class.

```php
$producer = Kafka::publishOn('[REMOTE_ADDRESS]', 'cleaners')
        ->withConfigOptions([
            'bootstrap.servers' => '[REMOTE_ADDRESS]',
            'metadata.broker.list' => '[REMOTE_ADDRESS]',
            'security.protocol' => 'SASL_SSL',
            'sasl.mechanisms' => 'PLAIN',
            'sasl.username' => '[API_KEY]',
            'sasl.password' => '[API_KEY]',
        ])
        ->withBodyKey("test", "test")
        ->send();
```

This throws an exception because these configOptions are not being passed to the Conf object from rdkafka. Probably because of this line:

`Junges\Kafka\Producers\Producer.php - Line 22`
```php
        $this->producer = app(KafkaProducer::class, [
            'conf' => $this->setConf($this->config->getProducerOptions()),
        ]);
```

Which limits the config passed to KafkaProducer::class to be only the ones returned from getProducerOptions().

Am i doing something wrong?
It works when i try to use php-rdkafka with the same configs.

**Comment by mateusjunges** [comment]
Hello @renatobcj, i forgot the `$this->customOptions` array merge for producer configs. Fixing it now. Thanks for your report.

**Comment by mateusjunges** [comment]
@renatobcj [#38ca04](https://github.com/mateusjunges/laravel-kafka/commit/38ca04c15b1feea10c33e9865377f712a1809d40) allows you to set custom options for producers config. Released on v1.1.3. Let me know if it fixes your issue.

**Comment by mateusjunges** [comment]
@renatobcj v1.2.0 should fix this problem. Please let me know if it's working well now.
